projects/angular-codegen/tsconfig.lib.json

```diff
@@ -12,7 +12,7 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "types": ["node", "sketchapp"],
-    "lib": ["dom", "es2018"]
+    "lib": ["dom", "es2019"]
```

Comment:
Did you make this change intentionally?

Reply:
Yes, I heavily use the `flatMap` and `flat` function in the codegen but it doesn't work without thes es2019.